script.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const menuList = document.querySelector('.menu-list');
+menuList.addEventListener('click', (event) => {
+    const activeElement = document.querySelector('.menu-active');
+    activeElement.classList.remove('menu-active');
+    const newActive = event.target;
+    newActive.classList.add('menu-active');
+})
+
+var slides = document.querySelectorAll('.slider-images');
+var currentSlide = 0;
+var isEnabled = true;
+
+function changeSlide(number) {
+  currentSlide = (number + slides.length) % slides.length;
+}
+
+function closeSlide(direction) {
+  isEnabled = false;
+  slides[currentSlide].classList.add(direction);
+  slides[currentSlide].addEventListener('animationend', function() {
+    this.classList.remove('slider-images--active', direction);
+  })
+}
+
+function showSlide(direction) {
+  slides[currentSlide].classList.add('slider-images--next', direction);
+  slides[currentSlide].addEventListener('animationend', function() {
+    this.classList.remove('slider-images--next', direction);
+    this.classList.add('slider-images--active');
+    isEnabled = true;
+  })
+}
+
+function nextSlide(number) {
```

Comment:
I'd recommend to use current/previous slide number instead of number. It'll be more convenient to read and easy to understand.